addons/sourcemod/scripting/ttt/ttt_shop.sp

```diff
@@ -714,6 +714,35 @@ bool ClientBuyItem(int client, char[] item, bool menu, bool free = false)
                     {
                         CPrintToChat(client, "%s %T", g_sPluginTag, "Item bought! (NEW)", client, g_iCredits[client], temp_item[Long], price);
                     }
+
+                    char sClientID[32], sRole[ROLE_LENGTH];
+                    TTT_GetRoleNameByID(TTT_GetClientRole(client), sRole, sizeof(sRole));
+
+                    ConVar cAddLogs = FindConVar("ttt_steamid_add_to_logs");
+                    ConVar cLogFormat = FindConVar("ttt_steamid_log_format");
+
+                    if (cAddLogs.BoolValue)
```

Comment:
To prevent errors, check if (g_)cAddLogs isn't null.

Reply:
Handled in my most recent commit.